libraries/apollo-http-cache/src/main/kotlin/com/apollographql/apollo3/cache/http/CachingHttpInterceptor.kt

```diff
@@ -84,15 +87,16 @@ class CachingHttpInterceptor(
           }
         }
       }
+
       else -> {
         error("Unknown HTTP fetch policy: $policy")
       }
     }
   }
 
-  private fun defaultPolicy(request: HttpRequest): String {
+  private fun getPolicy(request: HttpRequest): String {
```

Comment:
I'm guessing this is for the case of `CachingHttpInterceptor` is used without `HttpCacheExtensions`? Since `CachingHttpInterceptor` is public it's a technicality. But not sure either.